pkg/drivers/mysql/mysql.go

```diff
@@ -24,7 +24,7 @@ var (
 		`CREATE TABLE IF NOT EXISTS kine
 			(
 				id INTEGER AUTO_INCREMENT,
-				name VARCHAR(630),
+				name VARCHAR(630) character set ascii,
```

Comment:
```suggestion
				name VARCHAR(630) CHARACTER SET ascii,
```

Reply:
@brandond thanks for the prompt review, updated now